pkg/model/resources/nodeup.go

```diff
@@ -65,13 +65,13 @@ try-download-file() {
   local -r file="$2"
   local -r auth="$3"
   if [[ "$auth" == "" ]]; then
-		if curl -f --compressed -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10 "${url}"; then return 0; fi
-		if wget --compression=auto -O "${file}" --connect-timeout=20 --tries=6 --wait=10 "${url}"; then return 0; fi
-		if curl -f -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10 "${url}"; then return 0; fi
-		if wget -O "${file}" --connect-timeout=20 --tries=6 --wait=10 "${url}"; then return 0; fi
+        if curl -f --compressed -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10 "${url}"; then return 0; fi
```

Comment:
One reason to prefer tabs _here_ (for once, a reason) is that we're a little space constrained on AWS - the limit is 16KB, though we can gzip. OTOH if this actually becomes an issue I think feeding this through a filter that just trimmed all leading whitespace on a line would be the right thing to do, so I agree this makes sense.

Reply:
Yeah, I didn't modify it at first, but it causes the linter to complain - I can add a lint ignore tag if you like!